### PR TITLE
Vendor update for runtime-tools

### PR DESCRIPTION
**Pull request by umohnani8**
Pick up changes made to make setting ENV more efficient

Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@mrunalp @vrothberg PTAL

**Comment by umohnani8** [comment]
@saschagrunert PTAL.

File: go.mod [hunk 1]
```diff
@@ -68,8 +68,8 @@ require (
 	github.com/opencontainers/go-digest v1.0.0-rc1
 	github.com/opencontainers/image-spec v1.0.1
 	github.com/opencontainers/runc v1.0.0-rc7
-	github.com/opencontainers/runtime-spec v0.0.0-20180909173843-eba862dc2470
-	github.com/opencontainers/runtime-tools v0.6.0
+	github.com/opencontainers/runtime-spec v0.1.2-0.20190408193819-a1b50f621a48
+	github.com/opencontainers/runtime-tools v0.3.1-0.20190418135848-095789df6c2b
```
Comment (rhatdan) [inline go.mod]: Why are you vendoring an older runtime-tools?  Used to have v0.6.0 now you have v0.3.1?

Comment (umohnani8) [reply go.mod]: go mod is automatically generating that. I am not sure why but the changes are picked up in the vendor. @vrothberg do you know?

Comment (saschagrunert) [reply go.mod]: To discover the latest tag go calls currently

```
> git describe --first-parent --always --abbrev=0 --match "v[0-9]*.[0-9]*.[0-9]*" --tags
```

The argument `--first-parent` avoids collecting tags which are not done directly at the master branch.

Comment (vrothberg) [reply go.mod]: > go mod is automatically generating that. I am not sure why but the changes are picked up in the vendor. @vrothberg do you know?

To me, it seems that runtimes-tools has been pinned to a commit as `go mod` does not downgrade unless explicitly instructed to do so.

A `go get -u github.com/opencontainers/runtime-tools` updates to the latest version (i.e., `v0.9.0`). Same for the spec for getting `v1.0.1`.

Comment (umohnani8) [reply go.mod]: @vrothberg I need to pick up the most recent commit in master. Setting it to the latest version would pick up v0.9.0, which would not have the fixes we want to vendor.

Comment (vrothberg) [reply go.mod]: In that case, everything seems fine. @saschagrunert's comments explain why go-mod chooses the other prefix that we can safely ignore as the commit at the end is the important part.

Comment (saschagrunert) [reply go.mod]: I decided to follow up on this and proposed a change in golang (https://github.com/golang/go/issues/31673), let's see if it can be fixed.

**Pull request merged**